ubuntu/latest.Dockerfile

```
@@ -7,13 +7,151 @@
+    aws --version && \
+    docker --version && \
+    java -version && \
+    node --version && \
+    npm --version && \
+    npm run --version && \
+    npx playwright -V && \
+    sbt --version && \
+    yarn --version
+
+# Python virtual environment
+RUN python3 --version && \
```

Hadolint (2.12.0)

[info] Not following: File not included in mock.

(SC1091)